Resources/Locale/en-US/bed/cryostorage/cryogenic-storage.ftl

```diff
@@ -2,5 +2,5 @@
 ### Announcement
 
 earlyleave-cryo-job-unknown = Unknown
-earlyleave-cryo-announcement = {$character} ({$job}) { CONJUGATE-HAVE($entity) } entered cryogenic storage!
+earlyleave-cryo-announcement = {$character} ({$job}) has entered cryogenic storage!
```

Comment:
should i do it like this?
```suggestion
# {$entity} available for GENDER function purposes
earlyleave-cryo-announcement = {$character} ({$job}) has entered cryogenic storage!
```

Reply:
should make it easier for other languages to know it exists 👍